From c645fa7b1d87c0f47808e9b8bd5325820627a523 Mon Sep 17 00:00:00 2001
From: Andrii Osipov <aosipov@anaconda.com>
Date: Thu, 2 Jan 2025 18:08:14 -0800
Subject: [PATCH] Don't specify a pre-release in the numpy dependency

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 5e59da80..6b60de61 100644
--- a/setup.py
+++ b/setup.py
@@ -14,7 +14,7 @@ with open(os.path.join(here, "flaml/version.py")) as fp:
 __version__ = version["__version__"]
 
 install_requires = [
-    "NumPy>=1.17.0rc1",
+    "NumPy>=1.17",
     "lightgbm>=2.3.1",
     "xgboost>=0.90",
     "scipy>=1.4.1",
-- 
2.39.3 (Apple Git-146)

